ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,9 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    self.storage[self.current] = item
+    self.current += 1
+    self.current = self.current % len(self.storage)
```

Comment:
Hmm I don't necessarily know what this line is doing 🤔 
The first part is right. All you have to do next is check to see if the current is at capacity, if it is, reset current to 0